### PR TITLE
Building with Responsive Design #1

### PR DESCRIPTION
**Pull request by abmAsadullah**
## Building with Responsive Design
* Basic structure started
* Completed the project
* Make it responsive
* Create pull request

### CODE REVIEW

File: index.html
Comment (kevcap) [inline index.html]: - [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

 WOW, what project! ⭐


#### Hey @abmasadullah and @deikdesign , Is a pleasure to review such a good project! Congrats for the amazing work you guys already did, the project looks awesome! It breaks my heart 💔 not approve it right away, but I swear, we're almost done, there are just some minor changes that you guys have to implement to proceed:

  _Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.
  
## [REQUIRED CHANGES]
#### Clone REQUIREMENTS 👥 (Those adjusts to make your page really look like the real one)⤵
- [x] Those images should fill the whole space, please remove that blank space (check img below) ⤵
image
- [x] Align text with image (large screens) (📷 img1)
- [x] Remove horizontal bar (in small, medium) (📷 img2)
- [x] Add social icons (in small, medium and large screens)
- [x] Implement fixed navbar in small and medium screens.
- [x] Remove blank space at the bottom of the page (in small, medium, and large screens) (📷 img1)
#### Img1
image
#### Img2
image


#### Good practices REQUIREMENTS (Those changes to make you code like a pro 👽)⤵

- [x] Guys, next time check up the way you write some commits, they should be descriptive so the other programmer can see exactly what you changed with that commit. [Check this link](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/)
image
⬆ We can't exactly know what was changed on some of those commits

- [x] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */  
- [x] Your code is throwing some errors and warnings at the [HTML5 Validator](https://validator.w3.org/nu/#textarear), please fix them.
image

- [x] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
```  
      12 |   cursor: pointer;
      11 |   margin: 0;
       9 |   color: white;
       8 |   position: absolute;
       6 |   justify-content: space-between;
```

## [OPTIONAL CHANGES]
- [ ] Add a favicon to make it more similar to the original page;
- [ ] Put img folder and the css folder inside the assets folder.

## Summary
✨ Your page looks AWESOME! 👏
✨ Awesome CSS practices, congrats 😎
✨ Good job with that project, it looks really good for a first review 💥
✨ You guys got my attention, I'll keep my eyes on you! Keep up the good work! 🐱‍👓

## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀

  👨🏻‍💻 **TSE: Kevin Alves**

    

  📫 Have any question? Better call Kevin:

  <a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

    
  <sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

File: index.html
Comment (MuriloRoque) [reply index.html]: - [x] Please remove this commented piece of code.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: CHanges requested :red_circle: 

### Hello @abmasadullah and @deikdesign :wave: 

### Thank you for making almost all of the required changes from the previous TSE, but we still need to fix some of the things he asked:

- [x] You still have a horizontal scroll bar in all screen sizes, but I'm going to show you a screenshot of the small size, so you can see what is happening because of it (check the white space to the right):

Screenshot from 2020-09-10 18-24-35
<br>

- [x] You still have a commented piece of code, I made a comment below the last TSE's comment, please check below.

### Kindly make the requested changes and submit for another review, please :pray: 

### I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by kevcap — APPROVED** [review]
# [STATUS] PROJECT APPROVED 🍾

### _Great work, your clone looks almost identic,  you have also followed the best practices throughout the code, you guys got my attention and have my approval to merge this project and move on!_ 🥇💯

giphy4

## As always, I wish you all the very best and see you both at the top🚀


👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Pull request merged**
